tests/integration/golang/database/import_test.go

```diff
@@ -0,0 +1,246 @@
+
+	runs, err := inputRunFixtures.CreateExampleRuns(context.Background(), experiment, 5)
+	assert.Nil(s.T(), err)
+	s.runs = runs
+
+	// experiment 2
+	experiment, err = inputExperimentFixtures.CreateExperiment(context.Background(), &models.Experiment{
+		Name:           uuid.New().String(),
+		LifecycleStage: models.LifecycleStageActive,
+	})
+	assert.Nil(s.T(), err)
+
+	runs, err = inputRunFixtures.CreateExampleRuns(context.Background(), experiment, 5)
+	assert.Nil(s.T(), err)
+	s.runs = runs
+
+	appFixtures, err := fixtures.NewAppFixtures(helpers.GetInputDatabaseUri())
+	app, err := appFixtures.CreateApp(context.Background(), &database.App{
+		Base: database.Base{
+			ID:        uuid.New(),
+			CreatedAt: time.Now(),
+		},
+		Type:  "mpi",
+		State: database.AppState{},
+	})
+
+	dashboardFixtures, err := fixtures.NewDashboardFixtures(helpers.GetInputDatabaseUri())
+	_, err = dashboardFixtures.CreateDashboard(context.Background(), &database.Dashboard{
+		Base: database.Base{
+			ID:        uuid.New(),
+			CreatedAt: time.Now(),
+		},
+		AppID: &app.ID,
+		Name:  uuid.NewString(),
+	})
+
+	databaseSlowThreshold := time.Second * 1
+	databasePoolMax := 20
+	databaseReset := false
+	databaseMigrate := false
+	artifactRoot := "s3://fasttrackml"
+	input, err := database.MakeDBInstance(
+		helpers.GetInputDatabaseUri(),
+		databaseSlowThreshold,
+		databasePoolMax,
+		databaseReset,
+		databaseMigrate,
+		artifactRoot,
+	)
+	assert.Nil(s.T(), err)
+	output, err := database.MakeDBInstance(
+		helpers.GetOutputDatabaseUri(),
+		databaseSlowThreshold,
+		databasePoolMax,
+		databaseReset,
+		databaseMigrate,
+		artifactRoot,
+	)
+	assert.Nil(s.T(), err)
+
+	s.inputDB = input
+	s.outputDB = output
+
+	s.populatedRowCounts = rowCounts{
+		experiments:              3,
+		runs:                     10,
+		distinctRunExperimentIDs: 2,
+		metrics:                  40,
+		latestMetrics:            20,
+		tags:                     10,
+		params:                   20,
+		dashboards:               1,
+		apps:                     1,
+	}
+}
+
+func (s *ImportTestSuite) Test_Ok() {
+	defer func() {
+		assert.Nil(s.T(), s.inputRunFixtures.UnloadFixtures())
+		assert.Nil(s.T(), s.outputRunFixtures.UnloadFixtures())
+	}()
+
+	// source DB should have expected
+	validateRowCounts(s.T(), s.inputDB, s.populatedRowCounts)
+
+	// initially, dest DB is (mostly) empty
+	validateRowCounts(s.T(), s.outputDB, rowCounts{experiments: 1})
+
+	// invoke the Importer.Import() method
+	importer := database.NewImporter(s.inputDB, s.outputDB)
+	err := importer.Import()
+	assert.Nil(s.T(), err)
+
+	// dest DB should now have the expected
+	validateRowCounts(s.T(), s.outputDB, s.populatedRowCounts)
+
+	// invoke the Importer.Import method a 2nd time
+	err = importer.Import()
+	assert.Nil(s.T(), err)
+
+	// dest DB should still only have the expected
+	validateRowCounts(s.T(), s.outputDB, s.populatedRowCounts)
+
+	// confirm row-for-row equality
+	for _, table := range []string{
```

Comment:
here(and in other places as well) we use different approach without variable initialisation which if fine for such a one time logic.